packages/pma/pitop/pma/encoder_motor.py

```diff
@@ -286,15 +284,35 @@ def rotation_counter(self):
         the output shaft.
         """
 
-        dc_motor_rotation_counter = (
-            self.__motor_core.odometer() * self.__forward_direction
+        dc_motor_rotation_counter = self.__motor_core.odometer()
+        adjusted_dc_motor_rotation_counter = dc_motor_rotation_counter - (
+            self.__rotation_counter_offset
         )
         output_shaft_rotation_counter = round(
-            dc_motor_rotation_counter / self.MMK_STANDARD_GEAR_RATIO, 1
+            adjusted_dc_motor_rotation_counter / self.MMK_STANDARD_GEAR_RATIO, 1
         )
 
         return output_shaft_rotation_counter
 
+    @rotation_counter.setter
+    def rotation_counter(self, rotations):
```

Comment:
i think for our learning users it will be easier to understand a method motor.resetRotationCount() than a setter, which looks like a variable to them.  resetRotationCount could have basically the same implementation with a defaut of 0 and optional rotations number